bindings/core/tests/serialize_error.rs

```diff
@@ -12,9 +12,9 @@ fn custom_error_serialization() {
         serde_json::to_string(&error).unwrap(),
         "{\"type\":\"client\",\"error\":\"no healthy node available\"}"
     );
-    let error = Error::Wallet(WalletError::AccountNotFound("Alice".to_string()));
+    let error = Error::Wallet(WalletError::MissingBipPath);
```

Comment:
Try to replace with a similar error instead

Reply:
those are completely different errors (the former wasn't necessary anymore because there are no accounts anymore, and the new one comes with the fact that providing a BIP path is now optional, but can be necessary for certain operations.

Reply:
I have said it in the other comment, we do NOT care about the actual variant, this test is about custom serialisation

Reply:
gotcha